Airplane-tickets-UA.Data/City.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace Airplane_tickets_UA.Data
+{
+    public class City
+    {
+        public int Id { get; set; }
```

Comment:
no need to have both int ang GUID Ids, please consider removing one of them

Reply:
I need the ID only for the database, and GUID for displaying in the UI. GUID will be unique, assigned to the flight, and the ID can change depending on database operations.